gridy/Sources/Presentations/PlanBoard/TimelineLayout/ListArea/ListAreaView.swift

```diff
@@ -2,99 +2,149 @@
-                                    myIndex: index
+                    }
+                    .stroke(Color.gray, lineWidth: viewStore.columnStroke)
+                    
+                    // MARK: - ListArea Contents
+                    VStack(spacing: 0) {
+                        // MARK: - LayerIndex Section
+                        // TODO: - 이 액션 LayerIndex Area 나오면 옮기기. 지금은 temp.
+                        HStack {
+                            Button {
+                                viewStore.send(
+                                    .showLowerLayer
                                 )
+                            } label: {
+                                Text(">> 엎기")
+                            }
+                            .disabled(viewStore.showingLayers[0] == 0 && viewStore.showingLayers.count == 1)
+                            
+                            Spacer()
+                            
+                            Button {
+                                viewStore.send(
+                                    .showUpperLayer
+                                )
+                            } label: {
+                                Text("엎기 <<")
+                            }
+                            .disabled(viewStore.showingLayers.last == viewStore.map.count - 1)
+                        }
+                        .frame(height: viewStore.lineAreaGridHeight / 2)
+                        
+                        // MARK: - ListItem Section
+                        HStack(alignment: .top, spacing: 2) {
+                            ForEach(Array(zip(viewStore.showingLayers.indices, viewStore.showingLayers)), id: \.0) { forIndex, layerIndex in
+                                VStack(alignment: .leading, spacing: 0) {
+                                    let layer = viewStore.showingLayers[forIndex]
+                                    let showingAtFirst = (viewStore.showingLayers.count == 3 && forIndex == 0)
+                                    
+                                    // MARK: - LayerIndex Section
+                                    ZStack {
+                                        RoundedRectangle(cornerRadius: 4)
+                                            .foregroundStyle(Color.clear)
+                                        RoundedRectangle(cornerRadius: 4)
+                                            .stroke(Color.gray, lineWidth: 0.3)
+                                            .overlay(
+                                                Text(showingAtFirst ? "L\(layerIndex)": "Layer\(layerIndex)")
+                                            )
+                                            .font(.custom("Pretendard-Regular", size: 11))
+                                    }
+                                    .frame(width: viewStore.listColumnWidth[viewStore.showingLayers.count-1][forIndex], height: viewStore.lineAreaGridHeight / 2)
```

Comment:
여기 코드가 한줄에 안들어와서 개행하면 좋겠습니다~
```suggestion
                                    .frame(
                                    width: viewStore.listColumnWidth[viewStore.showingLayers.count-1][forIndex], 
                                    height: viewStore.lineAreaGridHeight / 2
                                    )
```

Reply:
개행 완료~!